resources/leiningen/new/luminus/keeframe/src/cljs/core.cljs

```diff
@@ -89,13 +89,11 @@
 ;; Initialize app
 (defn mount-components []
   (rf/clear-subscription-cache!)
-  (r/render [#'root-component] (.getElementById js/document "app")))
-
-(defn init! []
-  (ajax/load-interceptors!)
   (kf/start! {:debug?         true
-              :router         (routing/->ReititRouter routing/router)
+              :routes         routing/routes
```

Comment:
Implementation of kee-frame is now pretty much stolen from the code deleted in this PR. So I guess it's safe to use that instead? I could have another look to see if they are in fact identical.